### PR TITLE
Add the /status endpoint

### LINKED ISSUE #1936: API endpoint to get System Status
Can we get an endpoint that just gets the entire system status message.
For example, if all systems are operational the message the endpoint should return

```json
{
  "message": "All systems are operational"
}
```

### PR DESCRIPTION
**Pull request by adammbalogh**
Closes #1936

---

The API endpoint is ```/status```for getting all kinds of status messages. Right now it looks like this:
```json
{
  "system_status": "danger",
  "system_message": "The service experiencing a major outage"
}
```

(It still needs unit tests.)

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
Please don't put things in the foundation namespace.

File: app/Foundation/ViewObjects/SystemStatusViewObject.php [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace CachetHQ\Cachet\Foundation\ViewObjects;
+
+use CachetHQ\Cachet\Models\Component;
+use CachetHQ\Cachet\Foundation\Queries\SystemStatusQuery;
+
+class SystemStatusViewObject
```
Comment (GrahamCampbell) [inline app/Foundation/ViewObjects/SystemStatusViewObject.php]: View object isn't really right. It's not used in the context of views exclusively?

Comment (adammbalogh) [inline app/Foundation/ViewObjects/SystemStatusViewObject.php]: Yeah, it's not quite right. I'm thinking about how we should name this object.

Comment (GrahamCampbell) [inline app/Foundation/ViewObjects/SystemStatusViewObject.php]: We don't really need an object tbh. Just pass about an array.

**Comment by adammbalogh** [comment]
About foundation namespace. Sorry about that, I'm just learning the project with this PR. How we should structure my stuff, do you have an idea?

**Comment by GrahamCampbell** [comment]
I'll have to get back to you later. At work right now.

File: app/Foundation/ViewObjects/SystemStatusViewObject.php (hunk 1 above)
Comment (adammbalogh) [inline app/Foundation/ViewObjects/SystemStatusViewObject.php]: That can work but how about a transformer object maybe?

**Comment by jbrooksuk** [comment]
Replaced by #1968 :) much love to you for this @adammbalogh.

**Pull request closed without merging**